File: School Code/3. C++/School Code/SDEV325/helloworld.cpp
#include <iostream>
#include <ctime>

int main()
{
    // Get current date and time
    std::time_t now = std::time(nullptr);
    std::tm *local_time = std::localtime(&now);

    // Format date and time
    char time_buffer[80];
    std::strftime(time_buffer, sizeof(time_buffer), "%Y-%m-%d %H:%M:%S", local_time);

    // Print formatted message
    std::cout << "Hello, World!" << std::endl;
    std::cout << time_buffer << std::endl;
    std::cout << "Brian Walters (Developer)" << std::endl;
    std::cout << "Greetings, I am looking forward to SDEV 325." << std::endl;
    std::cout << "I have never played around with C++ yet but I am excited to learn." << std::endl;

    return 0;
}

File: School Code/1. Java/School Code/SDEV325/helloworld.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class helloworld {

  public static void main(String[] args) {
    // Get current date and time
    LocalDateTime now = LocalDateTime.now();
    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
    String formattedTime = formatter.format(now);

    // Print formatted message
    System.out.println("Hello, World!");
    System.out.println(formattedTime);
    System.out.println("Brian Walters (Developer)");
    System.out.println("Greetings, I am looking forward to SDEV 325.");
    System.out.println("I have never played around with C++ yet but I am excited to learn.");
  }
}